Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -207,42 +207,46 @@ public void ProcessPullRequest_EmptyCache_DoesNotProduceOutput()
             sut.ProcessPullRequest(cache);
 
             sut.UnchangedFilesPath.Should().BeNull();
+            logger.AssertInfoLogged("Analysis has 0 unchanged files out of 0 files in the base branch.");
         }
 
         [TestMethod]
         public void ProcessPullRequest_AllFilesChanged_DoesNotProduceOutput()
         {
-            using var context = new CacheContext(TestContext);
+            using var context = new CacheContext(this);
             foreach (var path in context.Paths)
             {
                 File.WriteAllText(path, "Content of this file has changed");
             }
             context.ProcessPullRequest();
 
             context.Sut.UnchangedFilesPath.Should().BeNull();
+            logger.AssertInfoLogged("Analysis has 0 unchanged files out of 3 files in the base branch.");
         }
 
         [TestMethod]
         public void ProcessPullRequest_NoFilesChanged_ProducesAllFiles()
         {
-            using var context = new CacheContext(TestContext);
+            using var context = new CacheContext(this);
             context.ProcessPullRequest();
 
             context.Sut.UnchangedFilesPath.Should().EndWith("UnchangedFiles.txt");
             File.ReadAllLines(context.Sut.UnchangedFilesPath).Should().BeEquivalentTo(context.Paths);
+            logger.AssertInfoLogged("Analysis has 3 unchanged files out of 3 files in the base branch.");
         }
 
         [TestMethod]
         public void ProcessPullRequest_SomeFilesChanged_ProducesOnlyUnchangedFiles()
         {
-            using var context = new CacheContext(TestContext);
+            using var context = new CacheContext(this);
             CreateFile(context.Root, "AddedFile.cs", "// This file is not in cache", Encoding.UTF8);
             File.Delete(context.Paths.First());       // This file was in cache, but doesn't exist anymore
             File.WriteAllText(context.Paths.Last(), " // This file was modified");
             context.ProcessPullRequest();
 
             context.Sut.UnchangedFilesPath.Should().EndWith("UnchangedFiles.txt");
             File.ReadAllLines(context.Sut.UnchangedFilesPath).Should().BeEquivalentTo(context.Paths[1]);  // Only a single file was not modified
+            logger.AssertInfoLogged("Analysis has 1 unchanged files out of 3 files in the base branch.");
```

Comment:
I'm not very happy with the text of the message itself. Let me know if you have better ideas

Reply:
How about "Incremental PR analysis:  {} out of {} files are unchanged."

Reply:
Final version: `Incremental PR analysis: {} files out of {} are unchanged.`